conf/config.neon

```diff
@@ -1043,6 +1043,8 @@ services:
 
 	-
 		class: PHPStan\Rules\UnusedFunctionParametersCheck
+		arguments:
+			reportExactLine: %featureToggles.bleedingEdge%
```

Comment:
Please introduce a new toggle for that.

Reply:
`reportPreciseLineForUnusedFunctionParameter` is there